### ISSUE TITLE
encodingFormat use CWL example, mismatch with Workflow section

### ISSUE THREAD

**Issue by stain** [ResearchObject/ro-crate#99]
**Describe the bug**

In https://www.researchobject.org/ro-crate/1.1-DRAFT/data-entities.html?highlight=commonwl#adding-detailed-descriptions-of-encodings example we see an example of `encodingFormat` being a URI rather than media type: 

```json
 {
    "@id": "1st-tool.cwl",
    "@type": "File",
    "contentSize": "120",
    "description": "An example Common Workflow Language File",
    "encodingFormat": ["text/plain", {"@id": "https://www.commonwl.org/v1.0/Workflow.html"}]
  },
    {
    "@id": "https://www.commonwl.org/v1.0/Workflow.html",
    "name": "Common Workflow Language (CWL) Workflow Description, v1.0.2",
    "@type": "Website"
  }
```

First of all (@mr-c might correct me) I think the better URI here would be CWL's permalink <https://w3id.org/cwl/v1.0/> as shown on <https://www.commonwl.org/v1.0/Workflow.html>

Secondly this use of `encodingFormat` here seems to be at odds with `programmingLanguage` in [Workflows section](https://www.researchobject.org/ro-crate/1.1-DRAFT/workflows.html#describing-scripts-and-workflows) - it is unclear which one of these should be used - or both?

**Suggested fix**

Easiest fix is to avoid mentioning workflows at the `encodingFormat` example - perhaps use a geopositional format or something instead.

Longer fix would be to reconcile the two, or at least mention `encodingFormat` under the Workflow section. Most programming languages don't have a registered [IANA media type](https://www.iana.org/assignments/media-types/) but would usually be a variant of `text/plain` (CWL is encoding format `text/x-yaml` - even YAML does not have a media type yet).

So perhaps <https://yaml.org/spec/1.1/> could be a better example, but generic YAML rather than CWL-that-happens-to-be-YAML.

**Comment by stain** [comment]
See also #154 which looks at adding profiles to data entities.

**Comment by stain** [comment]
This will be solved by #230 that replaces the workflow with a **data** entity.
